### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sunilk429**
1. Added setup for env.local
2. Added Link to docs for setting up realdevsquad.com locally

Date: 15-09-2024

Developer Name: Sunil Kumar

---

## Description

Updated setup for local.env and setting proxy locally by updating hosts file
### Documentation Updated?

- [x] Yes
- [ ] No

### Under Feature Flag

- [ ] Yes
- [x] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [ ] Yes
- [x] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

</details>

## Test Coverage
<details>
<summary>Screenshot 1</summary>

</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,16 +28,31 @@ Display the skills a user has and the endorsements created for adding new skills
...
+
+4. **Run the Development Server**:
+    ```sh
+    npm run dev
+    ```
+
+5. **Run Tests**:
+    ```sh
+    npm run test
+    ```
```
Comment (yesyash) [inline README.md:57]: - when writing a readme with instructions to set up the repository make sure that you take the user to the end i.e. a link where they can see the app running. 
here is a small but efficient example: 
    - [website status](https://github.com/Real-Dev-Squad/website-status?tab=readme-ov-file#setup-and-running-locally)

Comment (yesyash) [reply README.md:57]: - also do not add the steps to change etc hosts directly here, link to the `avoiding cors during development` document.

**Review by yesyash — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sunilk429) [reply README.md:57]: Thanks Yash for your feedback, I will update this

Comment (sunilk429) [reply README.md]: I have made changes, Please if you could review the changes

**Review by yesyash — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -28,16 +28,33 @@ Display the skills a user has and the endorsements created for adding new skills
...
+4. **Run the Development Server**:
+    ```sh
+    npm run dev
+    ```
+5. **Go to Website**:
+   https://dev.realdevsquad.com
+
+   ```sh
+   https://dev.realdevsquad.com
+   ```
```
Comment (yesyash) [inline README.md:53]: - remove this

File: README.md [hunk 3]
```diff
@@ -28,16 +28,33 @@ Display the skills a user has and the endorsements created for adding new skills
...
+      
+   
+7. **Run Tests**:
```
Comment (yesyash) [inline README.md:56]: - put this under a heading 

### Testing
- To run tests run the following command:
```sh
npm run test
```

**Review by yesyash — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (sunilk429) [reply README.md]: I have added changes, if you could please review Yash

**Review by yesyash — APPROVED** [review]

**Review by prakashchoudhary07 — APPROVED** [review]

**Pull request merged**